### DISCUSSION TITLE
Unable to access Repo

### DISCUSSION THREAD

**Discussion by DRXAquosus** [roxy-wi/roxy-wi#352] [Q&A] [answered]
I followed the instructions at https://roxy-wi.org/installation#rpm&gsc.tab=0 and added my creds to the roxy-wi.repo file.

Every time yum runs it comes up with:
Roxy-WI repo                                                                            245  B/s | 196  B     00:00
Errors during downloading metadata for repository 'Roxy-WI':
  - Status code: 404 for https://<UsernameRemoved>:<PasswordRemoved>@repo.roxy-wi.org/$ostype9/repodata/repomd.xml (IP: 104.21.55.204)
Error: Failed to download metadata for repo 'Roxy-WI': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried

Am I missing something?

Thanks

**Comment by Aidaho12** [comment, marked as answer]
Hello,
Change the $ostype to the el in the baseurl

**Reply by DRXAquosus** [reply]
To clarify.  
I changed this: repo.roxy-wi.org/$ostype$releasever/
To this: repo.roxy-wi.org/el$releasever/
and it worked after that.

Thanks
